lib/src/excel.dart

```diff
@@ -12,13 +12,36 @@ import 'util.dart';
 /// Service class that provides dart functions to interact with Excel.
 @Injectable()
 class ExcelDart {
-  static const _startAddress = 'A1';
-  static final _startAddressInt = 'A'.codeUnitAt(0);
+  static const _tableColumnStartAddress = 'A';
+  static const _tableRowStartAddress = 4;
```

Comment:
how come it starts at row 4?

Reply:
Please see [here](https://github.com/googleinterns/dv360-excel-plugin/pull/53#issue-448467208). Table starts on row 4, leaving some space to display underpacing options. 

If user doesn't check the underpacing checkbox, should we also display this, and shows 'highlight underpacing' as false, or should we hide this and move the table up?

Reply:
Looks good.

last thing, lets add the current pacing proportion into the schema